Extensions/view_on_dash.css

```diff
@@ -39,6 +39,10 @@
 	cursor:pointer;
 }
 
+.post_full .post_permalink {
```

Comment:
Make this line ``#view-on-dash-content .post_full .post_permalink {``, otherwise 
![image](https://cloud.githubusercontent.com/assets/1437010/10144584/c413801e-661e-11e5-95ac-af3f6a6df234.png) happens on the dashboard